ddocs/medic-client/views/contacts_by_last_visited/reduce

```diff
@@ -0,0 +1 @@
+_stats
```

Comment:
In theory we could implement something here which is more specific, just returning the max, which would return how much data to push down the line.

However,  `_stats` is native in CouchDB so it would be quite a bit faster to just generate the whole thing than do it in JS in terms of CPU time.

However however, we're probably only running this from PouchDB anyway, so that doesn't make a difference.

However however however, if we're only running it in PouchDB we also don't care about how much data gets pushed down the line.